bokehjs/src/coffee/models/transforms/interpolator.coffee

```diff
@@ -0,0 +1,85 @@
+_ = require "underscore"
+Transform = require "./transform"
+p = require "../../core/properties"
+
+class Interpolator extends Transform.Model
+
+  initialize: (attrs, options) ->
+    super(attrs, options)
+    @_x_sorted = []
+    @_y_sorted = []
+    @_sorted_dirty = true
+
+    @bind 'change', () ->
+      console.log 'Changed the configuration'
+      @_sorted_dirty = true
+
+  props: ->
+    return _.extend {}, super(), {
+      x: [ p.Any, '']
+      y: [ p.Any, '']
+      data: [ p.Any, null]
+    }
```

Comment:
Ok, I have adjusted this and have double checked the other transforms in the JS transforms dir to convert those as well.